azurelinuxagent/ga/update.py

```diff
@@ -345,6 +352,8 @@ def run(self, debug=False):
                         duration=duration,
                         message="Incarnation {0}".format(exthandlers_handler.last_etag))
 
+                self._send_heartbeat_telemetry(protocol)
```

Comment:
defined '_send_telemetry_heartbeat', but invoking '_send_heartbeat_telemetry'

Reply:
Thanks for catching this.

Reply:
could you do a dcr run with these changes? thanks

Reply:
@narrieta - Automation OK.